ImageD11/sinograms/dataset.py

```diff
@@ -47,9 +51,16 @@ def __init__(self,
                  dset = "dataset"):
         """ The things we need to know to process data """
 
+        
+        # it's better to determine the omegamotor, dtymotor and detector from the masterfile itself...
```

Comment:
Then have:
 ... ` omegamotor = None, dtymotor=None, detector=None `

`if thing is None: guess_thing()`

Reply:
If the info is supplied - the select the scan which have the motors.

Also handle the case of no dty motor.

Also handle the case of z scans vs y scans (etc, samry, samrx, diffry)